src/mm/dbcsr_mm.F

```diff
@@ -641,7 +641,7 @@ SUBROUTINE dbcsr_multiply_generic(transa, transb, &
       ! update statistics (we count marketing flops per MPI rank)
       dbcsr_mpi_statistics%last_mpi_ranks_used = numnodes
       marketing_flops = marketing_flops + &
-                        (2.0*(l_row - f_row + 1.0)*(l_col - f_col + 1.0)/numnodes)*(l_k - f_k + 1.0)
+                        REAL(2*(l_row - f_row + 1)*(l_col - f_col + 1))/REAL(numnodes)*REAL(l_k - f_k + 1)
```

Comment:
do we have a risk of int overflow here? The previous code did the multiplication using real, while I prefer to use int and then convert

Reply:
There was an overflow here time ago and that's why I changed to real. This is particularly true for long simulations.
This "marketing" flops is there only for statistics, it is not really useful. I like your change...

Reply:
ok, but then I have to do: `2.0*REAL(l_row - f_row + 1)*REAL(l_col - f_col + 1))/REAL(numnodes)*REAL(l_k - f_k + 1)`